.travis.yml

```diff
@@ -580,8 +476,6 @@ matrix:
           packages:
             - libreadline-dev
             - tcc
-            - gccgo
-            - golang-go
     - compiler: gcc
       dist: trusty
       env: PLATFORM=x32
```

Comment:
It looks like that cmocka build system is broken for x86-x32. You need to correctly propagate requirement of building library as static. For cmocka this can be done at cmake run time via option `-DWITH_STATIC_LIB=ON`. But it is broken. See attached patch for cmocka which fixes. [cmocka_static.patch.gz](https://github.com/pali/udftools/files/1671018/cmocka_static.patch.gz)

Then run `cmake -DWITH_STATIC_LIB=ON` but only in case `LDFLAGS` env variable contains `--static`. Same problem is for cross compilation for arm and powerpc.

Reply:
I see. Where did you get that patch? I would like to use it (e.g. wget it from somewhere and then apply it to sources) but I don't want wget it from this PR. 

I'll implement that fix as soon as possible.

Reply:
I wrote this patch myself from scratch.

Reply:
Oh, ok. I'll host patched version of cmocka somewhere and use that instead of official version.